hardcodes.R

```diff
@@ -260,7 +260,7 @@ syso_race_ethnicity_excl <- list(
 
 syso_veteran_pops <- list(
   "Veteran" = "Veteran",
-  "Non-Veteran/Unknown" = "NonVeteran"
+  "Non-Veteran" = "NonVeteran"
```

Comment:
I thought we wanted to combine non veteran and unknown? That's why we had this label. If we change it here, we have to change it throughout

Reply:
If you prefer Non-Veteran/Unknown, then we can make that work. I was able to make it work in my misc-fixes branch:
https://github.com/abtassociates/eva/blob/misc-fixes/system_composition_server.R#L155

![image](https://github.com/user-attachments/assets/1cd10eee-0e9c-4ee1-a470-afe50f1c3c03)